crates/eframe/Cargo.toml

```diff
@@ -90,6 +90,7 @@ serde = { version = "1", optional = true, features = ["derive"] }
 egui-winit = { version = "0.21.1", path = "../egui-winit", default-features = false, features = [
   "clipboard",
   "links",
+  "wayland",
```

Comment:
This probably should be an opt-in feature, instead of forcing it onto all `eframe` users.

In other words: add a `wayland` feature-flag to `eframe` with `wayland = ["egui-winit/wayland"]`

Reply:
Makes sense, fixed